README.md

```diff
@@ -1,2 +1,20 @@
 # HW01
-Edit the readme
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+# **Nikita Mehta**
+
+
+## **PhD Student in Neurobiology**  
+
+
+### **Email: nikitamehta@uchicago.edu**
+
+Nikita completed her undergraduate education at the University of Chicago. As an undergraduate research assistant in Dr. Brian Popko’s lab, she characterized roles of the Wnt signaling pathway in peripheral nerve regeneration using mouse models and explored potential therapeutics for patients with diabetic neuropathy. She also spent time at the Marine Biological Laboratory, studying the relationship between Alzheimer’s Disease and mitochondrial dysfunction under the mentorship of Dr. Elizabeth Jonas. She is interested in understanding the cellular mechanisms underlying neurodevelopmental disorders and neurodegeneration. She is involved with GRIT, Dean’s Council, and Graduate Council, and enjoys sunshine, running, biking, hiking, swimming, and any outdoor activity.
+
+
+<center>
+![](/Users/nikitamehta/Documents/Nikita.png)
```

Comment:
I think in order to get this to show up it needs to be in the same folder as the code you're working on :)